app/controllers/concerns/curation_concerns/embargoes_controller_behavior.rb

```diff
@@ -14,10 +14,9 @@ def index
 
     # Removes a single embargo
     def destroy
-      update_files = !curation_concern.under_embargo? # embargo expired
       EmbargoActor.new(curation_concern).destroy
       flash[:notice] = curation_concern.embargo_history.last
-      if update_files
+      if curation_concern.works_generic_work? && curation_concern.generic_files.present?
```

Comment:
In this case `curation_concern` could be a Work or a FileSet (fka GenericFile).  I'm not sure we really want to put it on FileSet, do we?